schemas/json/theme.json

```diff
@@ -1023,15 +1023,35 @@
 					"properties": {
 						"background": {
 							"description": "Sets the `background-color` CSS property.",
-							"type": "string"
+							"oneOf": [
+								{ "type": "string" },
+								{ "type": "object" }
+							],
+							"properties": {
+								"ref": {
+									"description": "A reference to another property value. e.g. `styles.color.text`",
+									"type": "string"
+								}
+							},
+							"additionalProperties": false
 						},
 						"gradient": {
 							"description": "Sets the `background` CSS property.",
-							"type": "string"
+							"oneOf": [
+								{ "type": "string" },
+								{ "type": "object" }
+							],
+							"$ref": "#/definitions/stylesProperties/properties/color/properties/background/properties",
```

Comment:
I'm not sure if this is the best way to reduce repetition! I wasn't sure where else to store the `ref` definition.